thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/function/WeekOverWeekRuleFunction.java

```diff
@@ -181,98 +133,7 @@ private RawAnomalyResultDTO getMergedAnomalyResults(List<RawAnomalyResultDTO> an
         }
       }
     }
-
-    String minConsecutiveSizeStr = props.getProperty(MIN_CONSECUTIVE_SIZE);
-    int minConsecutiveSize = 1;
-    if (StringUtils.isNotBlank(minConsecutiveSizeStr)) {
-      minConsecutiveSize = Integer.valueOf(minConsecutiveSizeStr);
-    }
-
-    return getFilteredAndMergedAnomalyResults(anomalyResults, minConsecutiveSize, bucketMillis,
```

Comment:
Can the merge algorithm handle a situation "treat this as an anomaly only if you see 4 consecutive of this type"? I thought merge algorithm only uses gap. So if we see 3 consecutive anomalies, we are going to generate an anomaly now if I understand this correctly